compose/docker-compose.override-lnd.yml

```diff
@@ -41,7 +41,7 @@ services:
     depends_on:
       - tor
       - lnd
-    command: litd --uipassword ${LIT_PASSWORD} --network=${NETWORK} --lnd-mode=remote --remote.lnd.rpcserver=127.0.0.1:${GUEST_LND_RPC_PORT:?} --remote.lnd.macaroonpath=/lnd/data/chain/bitcoin/${NETWORK}/admin.macaroon  --remote.lnd.tlscertpath=/lnd/tls.cert --httpslisten=0.0.0.0:8443 --loop.server.proxy=127.0.0.1:9050 --loop.loopoutmaxparts=25 --loop.maxlsatfee=1
+    entrypoint: litd --uipassword ${LIT_PASSWORD} --network=${NETWORK} --lnd-mode=remote --remote.lnd.rpcserver=127.0.0.1:${GUEST_LND_RPC_PORT:?} --remote.lnd.macaroonpath=/lnd/data/chain/bitcoin/${NETWORK}/admin.macaroon  --remote.lnd.tlscertpath=/lnd/tls.cert --httpslisten=0.0.0.0:8443 --loop.server.proxy=127.0.0.1:9050 --loop.loopoutmaxparts=25 --loop.maxlsatfee=1
```

Comment:
here makes more sense, I also noticed if your `LIT_PASSWORD` includes a special symbol like % or & it might cut off the remaining command and just send `entrypoint: litd --uipassword ${LIT_PASSWORD}`, but before fixing it, isn't there any other better way to share the password? 😅

Reply:
I guess the other way to feed the `uipassword` is by mounting a file with the `litd.conf`